src/org/zaproxy/zap/extension/wappalyzer/PopupMenuEvidence.java

```diff
@@ -71,6 +71,12 @@ public boolean isEnableForComponent(Component invoker) {
             	for (AppPattern p : app.getHtml()) {
             		this.addSubMenu("HTML", p.getPattern(), ExtensionSearch.Type.Response);
             	}
+            	for (Map<String,AppPattern> mp : app.getMetas()) {
+					for (Map.Entry<String, AppPattern> entry : mp.entrySet()) {
+						Pattern p = Pattern.compile(entry.getKey() + ".*" + entry.getValue().getPattern().pattern());
+	            		this.addSubMenu("META", p, ExtensionSearch.Type.Response);

```

Comment:
Surrounding code is mixed as well ... ??? ... As an example L62 and L73 have 12 spaces and a tab.

I propose it's left as is and fixed up whenever we get to an overall auto-formatting activity.